src/utils/ga-ecomm-helpers.js

```diff
@@ -0,0 +1,21 @@
+const filterEcommEvents = (obj) => {
+  const newObj = {};
+  Object.keys(obj).forEach((key) => {
+    if (key !== 'hitType' && key !== 'customTrackerId') {
+      newObj[key] = obj[key];
+    }
+  });
+  return newObj;
+};
+
+const isEcommEvent = event => !![
```

Comment:
`Array.prototype.find` is part of ES2015, and the browser support isn't great yet (no IE).

Normally I would use `Array.prototype.includes` in this situation, which would simplify the expression to: 

```js
const isEcommEvent = event => [
  'addTransaction',
  'addItem',
  'ecommSend',
  'ecommClear',
].includes(event.hitType);
```

But we should probably refrain from using newer ES features like these (Babel only handles syntax). We don't want to have our users download polyfills just so they can use `redux-beacon`. It seems pretty medieval, but we might be stuck with `indexOf`.

```js
const isEcommEvent = event => [
  'addTransaction',
  'addItem',
  'ecommSend',
  'ecommClear',
].indexOf(event.hitType) > -1;
```